pom.xml

```diff
@@ -46,7 +46,7 @@
     <jenkins.host.address />
     <slaveAgentPort />
     <!-- TODO fix KubernetesCloudTest todos once past 2.414 -->
-    <jenkins.version>2.401.1</jenkins.version>
+    <jenkins.version>2.414.2</jenkins.version>
```

Comment:
see comment in the code here -> https://github.com/jenkinsci/kubernetes-plugin/blob/2da8e2794884c5e844e8e47f1c91c98249ed355f/src/test/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloudTest.java#L78-L85